"We can't access your claims or appeals information" error discovery

nathangthomas opened the pull request.

nathangthomas committed:
"We can't access your claims or appeals information" error discovery

williamphelps13 commented on products/claim-appeal-status/engineering/my-va-dashboard-discovery.md:
```diff
@@ -0,0 +1,43 @@
+## Summary
+A logic error in the My VA dashboard prevents users from accessing their claims information when the Appeals API returns an error, even when the Claims API is functioning correctly.
+
+## Current Behavior
```
Could update this to be "Steps to Reproduce" as that is a bit more common and just make it a 5 item list that includes all the below steps (remove the 2 headings)

williamphelps13 commented on products/claim-appeal-status/engineering/my-va-dashboard-discovery.md:
```diff
@@ -0,0 +1,43 @@
+# Discovery: Claims and Appeals Access Issue on My VA Dashboard
```
With this being a pretty short document (unlike the date discrepancy document) I think when the review is done you could just copy and paste this markdown into a comment on the Github issue then you can share a link to that comment in the thread. But it was great to have a place to comment. You could just close this PR after. But want to also hear what you prefer of course.

nathangthomas commented on products/claim-appeal-status/engineering/my-va-dashboard-discovery.md:
```diff
@@ -0,0 +1,43 @@
+- Claims: Accessible
+- Appeals: 500 error
+
+*It is worth noting that the Appeals 500 error source for Staging User 220 is caused by a vets-api JSON schema validation failure. View thread on this issue [HERE](https://github.com/department-of-veterans-affairs/va.gov-team/issues/115204). 
```
The ticket for discovery around this bug is the link provided. I am unaware of a ticket for resolving this issue. This is something that Joshua Drumm picked up when I was moved to eventbus-gateway work.

nathangthomas committed:
Enhance documentation for My VA dashboard error handling

Expanded the summary and added steps to reproduce the issue with the My VA dashboard. Clarified the root cause and user impact while providing recommendations for improved error handling logic.
